packages/hardhat/README.md

```diff
@@ -55,3 +55,17 @@ npx hardhat verify <CONTRACT_ADDRESS>  <CONSTRUCTOR_ARGS> --network celo
 ```
 
 Check the file `hardhat.config.js` for Celo specific hardhat configuration.
+
+5. ABI Synchronization
+The project includes automatic ABI synchronization with your React frontend. ABIs are synced to `../react-app/src/abis/` during compilation.
+
+**Usage**
+- ABIs automatically sync on every `npx hardhat compile`
```

Comment:
the ABIs only sync automatically when you run `**yarn** run compile` or `npm run compile`